src/views/homepage/components/pre-footer/pre-footer.module.css

```
@@ -0,0 +1,67 @@
+.preFooter {
+  --container-max-width: 1140px;
+  --grid-template-columns: 4;
+  --grid-column-gap: 24px;
+
+  position: relative;
+  padding: 64px 24px;
+  background-image: radial-gradient(
+    at bottom,
+    var(--token-color-palette-neutral-100),
+    --token-color-palette-neutral-0
+  );
+
+  @media (--medium-up) {
+    --grid-template-columns: 12;
+    --grid-column-gap: 32px;
+  }
+}
+
+.container {
+  max-width: var(--container-max-width);
+  margin-right: auto;
+  margin-left: auto;
+  display: grid;
+  grid-template-columns: repeat(var(--grid-template-columns), 1fr);
+  grid-column-gap: var(--grid-column-gap);
+  grid-row-gap: 32px;
+  align-items: center;
+}
+
+.content {
+  grid-column: 1 / -1;
+
+  @media (--medium-up) {
+    grid-column: 2 / 6;
+  }
+}
+
+.description {
+  margin-top: 12px;
+  color: var(--token-color-foreground-primary);
+}
+
+.actions {
+  grid-column: 1 / -1;
+
+  @media (--medium-up) {
```

I think in dev-dot we want to avoid the media queries from `mktg-global-styles`, and instead use the dev-dot specific ones defined in https://github.com/hashicorp/dev-portal/blob/main/src/styles/custom-media.css:

```
/*
***
* DevDot
***
* The custom media below are helpers used in DevDot.
***
*/

@custom-media --dev-dot-mobile only screen and (max-width: 728px);
@custom-media --dev-dot-tablet only screen and (min-width: 729px) and (max-width: 1000px);
@custom-media --dev-dot-tablet-up only screen and (min-width: 729px);
@custom-media --dev-dot-desktop only screen and (min-width: 1001px);
```

I think in this case we'd want `--dev-dot-tablet-up` rather than `--medium-up`:

```suggestion
  @media (--dev-dot-tablet-up) {
```